src/core/server/graph/schema/schema.graphql

```diff
@@ -1240,6 +1240,22 @@ type StoryMessageBox {
   content: String
 }
 
+################################################################################
+## Embed Links
+################################################################################
+
+type EmbedLinksConfiguration {
+  """
+  twitterEnabled when true will enable twitter link embeds in the stream.
+  """
+  twitterEnabled: Boolean!
+
+  """
+  youtubeEnabled when true will enable youtube link embeds in the stream.
+  """
+  youtubeEnabled: Boolean!
```

Comment:
Suggestion: Rename fields to `twitter: Boolean!` and `youtube: Boolean!`. The "Enabled" part of the property can be inferred from the type.